dist/emoji-en-US.json

```
@@ -7152,7 +7179,9 @@
     "food",
     "japanese",
     "box",
-    "lunch"
+    "lunch",
+    "add_assets",
+    "update_assets"
```

```suggestion
    "assets"
```

a bit far fetched but eh.